bootstrap/kubeadm/config/manager/manager.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             - /manager
           args:
             - "--leader-elect"
-            - "--metrics-bind-addr=localhost:8080"
+            - "--diagnostics-address=:8443"
```

Comment:
Not against this, but, should we keep the default behavior the same as before this change for at least one release? That would give more time and a better deprecation cycle to our users who might rely on this default on their setups. And then make the default change you have here for 1.6.

Reply:
I think in any case as soon as we make the switch in YAMLs they have to either adopt their monitoring or use a different arg. So I'm not sure how much we gain by doing this 4 months later. But open to discuss

Reply:
I think it would be better to change the default in YAMLs and controllers at the same time to avoid additional confusion for users

Reply:
One addition, currently we bind to localhost:8080. This doesn't make it impossible but relatively hard to scrape metrics. So I wonder how many users are currently scraping from localhost:8080